dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/bnd.overrides

```diff
@@ -27,15 +27,9 @@ Import-Package: \
  com.ibm.websphere.ras, \
  com.ibm.websphere.ras.annotation, \
  com.ibm.ws.ffdc, \
- org.apache.cxf.*, \
+ org.apache.cxf.*;version="[2.6.2,2.6.3)", \
  *
  
-DynamicImport-Package: org.apache.cxf.bus;version="[2.6.2,2.6.3)", \
```

Comment:
Sadly, I don't know why I added this so I've removed it as it  was not in the original bindings-soap bnd.